go.mod

```diff
@@ -80,6 +80,10 @@ require (
 	modernc.org/sqlite v1.34.1
 )
 
+// Pinning as upgrade leads to error:
+// go: google.golang.org/grpc/gcp/observability@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
+replace google.golang.org/grpc/examples => google.golang.org/grpc/examples v0.0.0-20250219164421-42fc25a9b496
+
```

Comment:
You can also Pin to a fixed version in Makefile https://github.com/temporalio/temporal/blob/fd35dc18beec8a05a7b1b50adc4c643e4a4a7d5a/Makefile#L120 and just run make update-depencencies

Reply:
I didn't know about that one; I've changed the PR to use that now!